SalesforceCommerceCloud/pwa-kit
# Update translations

# Description
Update non-English translations from translation team
Deployed version: https://scaffold-pwa-test-env-2.mobify-storefront.com/global/fr-FR/




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

Here's how to see a different locale:
1. Go to this site where we've staged the translations: https://scaffold-pwa-test-env-2.mobify-storefront.com/
2. Scroll down to the footer and find the locales dropdown
3. Select a different locale

FYI the translation team has tested the translations and given us their approval.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
